service/src/main/java/bio/terra/landingzone/service/landingzone/azure/LandingZoneService.java

```diff
@@ -117,14 +123,19 @@ public void deleteLandingZone(String landingZoneId) {
     throw new LandingZoneDeleteNotImplemented("Delete operation is not implemented");
   }
 
-  public LandingZoneResourcesByPurpose listResourcesWithPurposes(
```

Comment:
We also need a way to call `listResourcesWithPurposes` using the tenant, subscription, and resource group - basically what we call the `AzureCloudContext`.  This will be used when WSM creates resources in the LZ. Few options to consider:
 1.- Keep `listResourcesWithPurposes(AzureCloudContext azureCloudContext)` in addition to your new method.
 2.- Add `listResourcesWithPurposes(tenant, sub, rg)` in addition to your new method.
 3.- Create a new method that returns the LandingZoneIds per the given information: `listLandingZoneIds(tenant, sub, rg)`

I don't have a strong preference for any of these, and all have some trade offs: 1 and 2 are simpler, but hide the many to many semantics of the model, which could create problems down the road. 3 more complex for the caller, but in sync with the model.  To get going, any approach would be fine with me as long as we are aware of the implications.

Reply:
Why do we need this version with landing zone target parameters?

Reply:
Calls to WSM to create resources in the LZ won't include the landing zone id. Eventually, we will make a change in WSM to store the landingzone id in the WSM cloud context. But not initially.

Reply:
Why?




































[TOAZ-194](https://broadworkbench.atlassian.net/browse/TOAZ-194)
**Implement REST API for listing of resources** in a landing zone and their purpose